Handbook: Add structure to performance reviews (#2523)
FlowFuse/website: src/handbook/peopleops/performance-review.md

While this performance review is written down, each manager should've communicated
feedback regularly to minimize the chance for surprises during the written review.

## Structure

> You can find a template document in Google Drive [here](https://docs.google.com/document/d/1E_Ni5kIcgRAvx9ZxFECOxJkFzzvDoUSpb5YfzKd1zCk/).

Reviewer: Nit: here as link text I think is a bit lazy

Reviewer: ```suggestion
A [template document](https://docs.google.com/document/d/1E_Ni5kIcgRAvx9ZxFECOxJkFzzvDoUSpb5YfzKd1zCk/) is available to structure the review.
```

Please be sure to duplicate ***and move*** the document before filling it out as to not share employee performance reviews with others unintentionally.

The template consists of the following:

### Values

Each of the Company Values will be rated as one of the following:

- **Exceeds:** Employee consistently goes above and beyond in this area.
- **Achieves:** Employee meets expectations for their seniority in this area.
- **Room for Growth:** This is an area where the employee could improve.

Details and examples should be provided with each instance. It is also important that employees are not disheartened
if given "Room for Growth" feedback, in fact, managers are encouraged to assign this rating to at least one of the
company values, although this should not be forced.

It is almost impossible to achieve/exceed all values perfectly, sometimes particular employees are more focus on
customer/community engagement, others may be more focussed on specific technical requirements. This is normal and should be embraced.

### Highlights

A chance to focus on the best things that the employee has contributed to FlowFuse since their last review (or first day

> if this is their first review). Highlights should come from both employees and their managers, and are a chance to reflect

Reviewer: > Highlights should come from both employees and their managers

How do others inform a manager of highlights of their peer? It seems like the manager should be observant?

Author: Good point - I had thought about 360 Feedback too, but didn't want to introduce too much structure. We do also have the [Slack bots](https://flowfuse.com/handbook/company/communication/#feedback-and-thanks) for this, but I don't think I've ever seen anyone utilise them.

Reviewer: Point of feedback for the review! :)
